src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -30,7 +31,16 @@ We help you get to the heart of your system by offering two quick and easy ways
         src={nrqlconsole}
```

Comment:
Not part of this PR, but I suggest using a non-cropped screenshot here. The image is hard to parse and the crop makes it seem really "busy" since the background (the charts) have no visual context.

Reply:
Yes, I agree. I am waiting on better marketing imaging. This is just a placeholder @bradleycamacho .